keel-retrofit-test-support/src/main/kotlin/com/netflix/spinnaker/keel/retrofit/model/ModelParsingTestSupport.kt

```diff
@@ -70,16 +65,3 @@ abstract class BaseModelParsingTest<out T> {
   }
 }
 
-fun Request.jsonResponse(
```

Comment:
Where did this go?

Reply:
it wasn't used anywhere as far as I can see